packages/react/src/_popover/Popover.tsx

```diff
@@ -12,8 +12,12 @@ import { Box } from '../box';
 import { forwardRefWithAs, tokens } from '../core';
 
 export type PopoverProps = PropsWithChildren<{
-	visibility?: 'visible' | 'hidden';
+	/** Fix the height of the popover to match the height of the content. */
+	fixHeightAsContentHeight?: boolean;
```

Comment:
This description is insufficient documentation. I still have no idea when I should set this to true and when I should set this to false.